batchflow/dataset.py

```diff
@@ -75,7 +75,6 @@ def __init__(self, index, batch_class=Batch, *args, preloaded=None, cast_to_arra
         self._attrs = None
         kwargs['_copy'] = kwargs.get('_copy', copy)
         self.n_splits = None
-        self.train, self.test = None, None
```

Comment:
Why do you think this is a better approach?

Reply:
Looks weird to reinitialize attributes, that have already been initialized in the parent class `Baseset`

Reply:
I return it as it was initially and plug `pylint: disable=attribute-defined-outside-init` stub.
In the code we assign values to `self.train / self.test / self.validation` attributes that WERE defined in the `init`.
It looks like pylint false positive